packages/deployment/upgrade-test/Makefile

```
@@ -0,0 +1,7 @@
+REPOSITORY = agoric/upgrade-test
+
+build:
+	docker build --progress=plain -t $(REPOSITORY):latest -f Dockerfile upgrade-test-scripts
+
+run:
+	docker run -it -e "DEST=1" -v "$$(PWD):/workspace" --entrypoint bash $(REPOSITORY):latest
```

This breaks for me in linux, while it works fine in mac:

```sh
make run
docker run -it -e "DEST=1" -v "$(PWD):/workspace" --entrypoint bash agoric/upgrade-test:latest
/bin/sh: 1: PWD: not found
root@d58ce43f96fe:/usr/src/agoric-sdk# ls /workspace
ls: cannot access '/workspace': No such file or directory
```

```sh
make run
docker run -it -e "DEST=1" -v "$(PWD):/workspace" --entrypoint bash agoric/upgrade-test:latest
root@75308157bf15:/usr/src/agoric-sdk#
```

Looks like when it resolves it's not portable across shells:

```sh
echo $SHELL; echo $(PWD)
/bin/bash
bash: PWD: command not found
```
```sh
echo $SHELL; echo $(PWD)
/bin/zsh
/Users/raphs/Code/agoric-sdk/packages/deployment/upgrade-test
```

Perhaps we should use:
```suggestion
	docker run -it -e "DEST=1" -v "$${PWD}:/workspace" --entrypoint bash $(REPOSITORY):latest
```